packages/react-instantsearch-dom/src/components/__tests__/SearchBox.js

```diff
@@ -25,6 +25,16 @@ describe('SearchBox', () => {
     instance.unmount();
   });
 
+  it('applies its default props with custom inputId', () => {
+    const instance = renderer.create(
+      <SearchBox inputId="search-box" refine={() => null} />
+    );
+
+    expect(instance.toJSON()).toMatchSnapshot();
```

Comment:
this test should probably only check the id is set

Reply:
Just updated to match the testing style of #3068!